iceberg/iceberg-common/src/main/java/org/apache/gravitino/iceberg/common/ops/IcebergCatalogWrapper.java

```diff
@@ -215,6 +216,11 @@ public LoadViewResponse updateView(TableIdentifier viewIdentifier, UpdateTableRe
   }
 
   public LoadViewResponse loadView(TableIdentifier viewIdentifier) {
+    // Trino Analyzer try to get view from Iceberg catalog, return NoSuchViewException to keep
+    // the compatibility with Trino.
+    if (!(catalog instanceof ViewCatalog)) {
+      throw new NoSuchViewException("Catalog %s doesn't support view", catalog.name());
```

Comment:
Is it better to throw an unsupported exception here?

Reply:
From the side of Gravitino, it's reasonable to throw `unsupported exception` but this will cause Trino query failed, because it will try to get view from Iceberg REST server in analyzer, please refer to https://github.com/trinodb/trino/blob/master/core/trino-main/src/main/java/io/trino/sql/analyzer/StatementAnalyzer.java#L2279

Reply:
Why can't we fail the query if it is not supported? For example, if we write a query with some unsupported features, shall we fail the query, or doing some implicit conversion?

Reply:
This will cause the simple SQL to fail in Trino like `select * from xx`.  The latest Trino code adds a configuration to enable compatibility with REST server that does not support view operations. But for the old trino releases, it couldn't work with Gravitino.  This PR moves the compatibility work from client to server, I agree that this is a workaround solution, is more general solution is https://github.com/apache/gravitino/issues/6395 but it benefits the cooperation with Trino and Gravitino IRS.